### PR TITLE
MGMT-7711: Show 'Technology preview' badge near the title of Assisted installer wizard title

### PR DESCRIPTION
**Pull request by jgyselov**
https://issues.redhat.com/browse/MGMT-7711

before:
image
after:
image

### CODE REVIEW

File: src/ocm/components/clusters/ClusterPage.tsx [hunk 1]
```diff
@@ -87,7 +88,13 @@ const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
           <ClusterBreadcrumbs clusterName={cluster.name} />
           <PageSection variant={PageSectionVariants.light}>
             <TextContent>
-              <Text component="h1">Install OpenShift with the Assisted Installer</Text>
+              <Text component="h1" className="assisted-ui-display-inline">
```
Comment (jkilzi) [inline src/ocm/components/clusters/ClusterPage.tsx:91]:
```suggestion
              <Text component="h1" className="pf-u-display-inline">
```
@jgyselov does this achieve the same?

File: src/ocm/components/clusters/ClusterPage.tsx [hunk 2]
```diff
@@ -87,7 +88,13 @@ const ClusterPage: React.FC<RouteComponentProps<MatchParams>> = ({ match }) => {
...
+                Install OpenShift with the Assisted Installer
+              </Text>
+              <TechnologyPreview
+                className="margin-left-md"
```
Comment (jkilzi) [inline src/ocm/components/clusters/ClusterPage.tsx:95]:
```suggestion
                className="pf-u-ml-md"
```

@jgyselov does this achieve the same?

**Review by jkilzi — CHANGES_REQUESTED** [review]

File: src/common/styles.css [hunk 3]
```diff
@@ -2,16 +2,3 @@
 .assisted-ui-float-right {
   float: right !important;
 }
```
Comment (jkilzi) [inline src/common/styles.css:4]: Can we get rid of this one too?

Comment (jgyselov) [reply src/common/styles.css]: I'm not sure what the _pf-u-float-right does not work in all flows_ refers to...

Comment (jgyselov) [reply src/common/styles.css]: But it doesn't seem we are using the `inlineRight` option anywhere so maybe we could get rid of this option entirely?

Comment (jkilzi) [reply src/common/styles.css]: Any chance you can test it quickly?

Comment (jgyselov) [reply src/common/styles.css]: Tested. Looked fine to me.

**Review by jkilzi — APPROVED** [review]

**Pull request merged**